web/api/routes.go

```diff
@@ -46,6 +46,18 @@ func (ar *Router) initRoutes() {
 		ar.Token(model.TokenTypeAccess, nil),
 		negroni.Wrap(ar.RequestInviteLink()),
 	)).Methods("POST")
+	auth.Path(`/{invite:invite/?}`).Handler(negroni.New(
+		ar.Token(model.TokenTypeAccess, nil),
```

Comment:
does it mean every user has access to the list of invites?

Reply:
Moved to the `admin` router